src/components/Navbar/Navbar.jsx

```diff
@@ -1,13 +1,20 @@
 import React, { Component } from 'react';
 import './Navbar.css';
-
+import slack from './slack.png'
+import youtube from './youtube.png'
+import github from './github.png'
+import twitter from './twitter.jpg'
 class Navbar extends Component {
     render() {
         return (
             <div className="topnav">
+                <img src={slack} className="slack"/>
                 <a href="https://join.slack.com/t/firstcontributors/shared_invite/enQtNjkxNzQwNzA2MTMwLTVhMWJjNjg2ODRlNWZhNjIzYjgwNDIyZWYwZjhjYTQ4OTBjMWM0MmFhZDUxNzBiYzczMGNiYzcxNjkzZDZlMDM"  target="_blank" rel="noopener noreferrer">Slack</a>
+                <img src={youtube} className="youtube"/>
                 <a href="https://www.youtube.com/channel/UCMXNFxCvyH5LhUwEcmY8qGQ" target="_blank" rel="noopener noreferrer">YouTube</a>
+                <img src={github} className="github"/>
                 <a href="https://twitter.com/1stContribution" target="_blank" rel="noopener noreferrer">Twitter</a>
+                <img src={twitter} className="twitter"/>
                 <a href="https://github.com/firstcontributions/first-contributions" target="_blank" rel="noopener noreferrer">GitHub</a>
```

Comment:
Looks like you swapped twitter and github.
See the icons https://github.com/firstcontributions/firstcontributions.github.io/issues/10#issuecomment-633080776

Reply:
I have done those changes but I don't know why they are not reflecting back

Reply:
So as you a pro in contribution can I ask for help from you

Reply:
can you please help with my issue

Reply:
You can make changes to the pull request by pushing commit to the same branch it's created from. In this case, ` mohak852:source`

Make sure you've pushed the changes.